src/subgame.cpp

```diff
@@ -223,15 +221,31 @@ std::string getWorldGameId(const std::string &world_path, bool can_be_legacy)
 	return conf.get("gameid");
 }
 
+std::string getWorldPathEnv()
+{
+	char *world_path = getenv("MINETEST_WORLD_PATH");
```

Comment:
Be aware that this may not work with all Windows configurations:
https://msdn.microsoft.com/library/windows/apps/jj606124.aspx